src/main/java/com/exasol/projectkeeper/validators/changelog/ChangelogFileValidator.java

```diff
@@ -0,0 +1,44 @@
+package com.exasol.projectkeeper.validators.changelog;
+
+import java.nio.file.Path;
+import java.util.Collections;
+import java.util.List;
+
+import com.exasol.errorreporting.ExaError;
+import com.exasol.projectkeeper.ValidationFinding;
+import com.exasol.projectkeeper.validators.AbstractFileContentValidator;
+import com.exasol.projectkeeper.validators.VersionLister;
+
+//[impl->dsn~verify-changelog-file~1]
+public class ChangelogFileValidator extends AbstractFileContentValidator {
+    private final Path projectDirectory;
+
+    /**
+     * Create a new instance of {@link ChangelogFileValidator}.
+     * 
+     * @param projectDirectory project's root directory
+     */
+    public ChangelogFileValidator(final Path projectDirectory) {
+        super(projectDirectory, Path.of("doc/changes/changelog.md"));
+        this.projectDirectory = projectDirectory;
```

Comment:
If you also send this to superclass, maybe you just save it in the superclass and use as a protected variable?

Reply:
in the super class I don't save it. So I think it's better to save it here.